netbox/templates/ipam/l2vpn.html

```diff
@@ -59,7 +59,7 @@ <h5 class="card-header">Terminations</h5>
       </div>
       {% if perms.ipam.add_l2vpntermination %}
         <div class="card-footer text-end noprint">
-          <a href="{% url 'ipam:l2vpntermination_add' %}?l2vpn={{ object.pk }}&return_url={{ object.get_absolute_url }}" class="btn btn-primary btn-sm">
+          <a href="{% url 'ipam:l2vpntermination_add' %}?l2vpn={{ object.pk }}&return_url={{ object.get_absolute_url }}" class="btn btn-primary btn-sm{% if not object.can_add_termination %} disabled{% endif %}"{% if not object.can_add_termination %} aria-disabled="true"{% endif %}>
```

Comment:
Can we simplify this a bit to avoid repeating the `{% if %}` block? I know the HTML is a little ugly but it eliminates the second evaluation. Alternatively, we could repeat the entire button element for better readability (once enable and once disabled).

```suggestion
          <a href="{% url 'ipam:l2vpntermination_add' %}?l2vpn={{ object.pk }}&return_url={{ object.get_absolute_url }}" class="btn btn-primary btn-sm{% if not object.can_add_termination %} disabled" aria-disabled="true{% endif %}">
```